README.md

```diff
@@ -156,6 +156,11 @@ Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning,
 * How do you organize your code? (module pattern, classical inheritance?)
 * What's the difference between host objects and native objects?
 * Difference between: `function Person(){}`, `var person = Person()`, and `var person = new Person()`?
+  - This is a function declaration that assigns a name to a function
```

Comment:
wg linku, który załączyłeś to jest "A Function Declaration defines a named function variable without requiring variable assignment. "